frappe/public/js/frappe/views/reports/report_view.js

```diff
@@ -1572,6 +1572,14 @@ frappe.views.ReportView = class ReportView extends frappe.views.ListView {
 							},
 						];
 					}
+					if (frappe.boot.lang !== "en") {
+						extra_fields.push({
+							fieldtype: "Check",
+							fieldname: "translate_values",
+							label: __("Translate values"),
+							default: 1,
```

Comment:
Should the default remain false? (not sure, just wondering because people may be used to just exporting and not realise there's a new checkbox here)

Reply:
I was wondering the same thing because, in some cases, users will export data just to further import back ... But, since we are checking if current language is not "English", I think that this default "True" value works well.

Reply:
Yep my only thought was people may not notice the new checkbox, that's why thought default behaviour should stay the same

Reply:
I think that "natural" behavior should to be translating, since "natural" reason for exporting is ... "exporting", not importing data again later ... which is -apparently- the only motif to preserve original" data. Non-english users expects "transparent" language workflow.  Importing-exporting data is usually done by medium-advanced users, that easily will be aware of new option ...

Anyway, maybe I am missing something. I can adapt it as your preference.

Thanks @akhilnarang (and sorry, I can't speak english fluently and I know my messages should be "translated" too sometimes xD.)

Reply:
Makes sense, will merge as is. Can change later if required then.

Reply:
We are running a v14 system were hundreds of non-advanced non-english users export for re-import on a regular basis. 🙈

Reply:
Kill'em ;)

Options:
- Revert
- Don't backport to v14 (since the world was running until this commit ...)
- Some kind of configurable setting (system settings, or user ...) . Even it would be connected to https://github.com/frappe/frappe/pull/27178